js/templates/userPage/store.html

```diff
@@ -27,7 +27,7 @@ <h1 class="txCtr">Loading...</h1>
 <% } else if (!ob.isFetching) { %>
   <div class="flex">
     <div class="col storeFilters <% if (!ob.listingCount) print('disabled') %>">
-      <div class="clrP clrBr border padMd clrT">
+      <div class="clrP clrBr border padMd clrT clrSh2 searchFilter">
```

Comment:
If the searchFilter class only adds the rounded corner, this div should probably get the contentBox class instead. That will add the corner and the border, so you could remove border and searchFilter.

It also sets the text size to tx4, so that class could be removed from the Shipping text inside it.

The contentBox class is intended to be used for all of the boxes in the design that share the same basic properties, I think there are some places we forgot to use it.

If we want all of those elements to stay consistent, and change together if we ever change the design (like to tweak the corner radius), then we should use the contentBox class. If we think one of those elements should stay the same if the others change, then we could use a custom class.